.github/workflows/flaky_tests.yml

```diff
@@ -0,0 +1,14 @@
+name: Build and run flaky Tests openDAQ
```

Comment:
So, let's just name this something other  than "Flaky Tests" :) Something like, "Nightly full build and test".